integration/test-integration/src/test/resources/dockerCompose/git-service.yaml

```
@@ -1,6 +1,6 @@
 
   gitlab:
-    image: "gitlab/gitlab-ee:latest"
+    image: "gitlab/gitlab-ce:latest"
```

Shall we use a specific version
```suggestion
    image: "gitlab/gitlab-ce:14.7.3-ce.0"
```

---

+1